static/bootstrapalert.js

```diff
@@ -14,16 +14,23 @@ const bootstrap5Alert = ({ message, title, actionText, cb, color }) => {
   if(color === undefined) {
     color = 'danger';
   }
+  if(color == 'danger') {
```

Comment:
I've added similar changes to my own branch to support multiple icons... Can you merge them with my changes?